pontoon/base/static/js/translate.js

```diff
@@ -4431,6 +4433,33 @@ Pontoon.user = {
   forceSuggestions: $('#server').data('force-suggestions') === 'True' ? true : false,
   manager: $('#server').data('manager'),
   localesOrder: $('#server').data('locales-order') || {},
+  canTranslate: function() {
+    // Same as Pontoon.user.canTranslate()
```

Comment:
This comment isn't helpful since this is effectively `user.canTranslate()`. This says that A is A. :-P I think you can simply remove it.